test_data/generated_test_data/ontologyR2RV2/knoraApiOntologySimple.jsonld

```diff
@@ -1075,12 +1090,12 @@
                 "@id": "knora-api:hasFile"
             },
             "rdfs:comment": "Connects a Representation to an image file",
-            "@type": "owl:DatatypeProperty",
+            "@type": "owl:ObjectProperty",
```

Comment:
**question(potentially-blocking):** I'm really confused about this change... why did that change?